File: Pascal/1012/1012.pas
uses math;
var
	a, b, c : double;
	pi : double;
	triangulo, circulo, trapezio, quadrado, retangulo : double;
begin
	pi := 3.14159;

	readln(a, b, c);

	triangulo := (a * c) / 2;
	circulo := pi * Power(c, 2);
	trapezio := ((a + b) * c) / 2;
	quadrado := b * b;
	retangulo := a * b;

	writeln('TRIANGULO: ', triangulo:3:3);
	writeln('CIRCULO: ', circulo:3:3);
	writeln('TRAPEZIO: ', trapezio:3:3);
	writeln('QUADRADO: ', quadrado:3:3);
	writeln('RETANGULO: ', retangulo:3:3);
end.

File: C++/1012/1012.cpp
#include <iostream>
using namespace std;

int main() {
	double a, b, c, TRIANGULO, CIRCULO, TRAPEZIO, QUADRADO, RETANGULO;

	cin >> a >> b >> c;

	TRIANGULO = (a * c) / 2;
	CIRCULO = 3.14159 * (c*c);
	TRAPEZIO = ((a + b) * c) / 2;
	QUADRADO = b * b;
	RETANGULO = a * b;
	
	printf("TRIANGULO: %.3lf\nCIRCULO: %.3lf\nTRAPEZIO: %.3lf\nQUADRADO: %.3lf\nRETANGULO: %.3lf\n", TRIANGULO, CIRCULO, TRAPEZIO, QUADRADO, RETANGULO);
}
